src/components/ScrollLock/tests/ScrollLock.test.tsx

```diff
@@ -10,13 +10,18 @@ describe('ScrollLock', () => {
         showScrollLock: true,
       };
 
+      setScollLockFalse = () => {
+        this.setState({showScrollLock: false});
+      };
```

Comment:
Don't we want to avoid setting state in tests?

Reply:
Changing state within a component through user interactions is ok, however, we want to avoid explicitly setting state with the enzyme API. However, in this case, we may be able to simplify the test by unmounting the wrapper, I'll give that a try.